db/seeds/development.rb

```diff
@@ -0,0 +1,26 @@
+# db/seeds/common.rb always runs before this, so we can assume
+# roles etc defined there exist already.
+
+# Add a nice, memorable, boring set of users for local development
+%w[user admin superadmin].each do |username|
+  User
+    .create_with(
+      name: username.capitalize,
+      email: "#{username}@example.com",
+      password: "passwordNZOI",
+      confirmed_at: Time.current
+    ).find_or_create_by!(
+      username: username
+    )
+end
+
+User.find_by!(name: "admin").update!(roles: Role.find_by!(name: "admin"))
+User.find_by!(name: "superadmin").update!(roles: Role.find_by!(name: "superadmin"))
```

Comment:
```suggestion
User.find_by!(username: "admin").update!(roles: [Role.find_by!(name: "admin")])
User.find_by!(username: "superadmin").update!(roles: [Role.find_by!(name: "superadmin")])
```

This currently fails to find the users since we're using capitalized names, and roles should be an array.

Reply:
Hmm but worrying I didn't catch that, but fixe dnow.